ckanext/bcgov/templates/package/read.html

```diff
@@ -119,7 +119,7 @@
   {% block package_additional_info %}
     <h3>Additional Information</h3>
     <div id="additional-information">
-      {% if pkg.object_name and pkg.resources and pkg.resources[0].resource_storage_location == 'BCGW Data Store' %}
+      {% if pkg.object_name and pkg.resources %}
```

Comment:
So, this is the change that should be visible. I'll find a dataset with a non-"BCGW Data Store" resource[0] and see if the change takes place.